src/generic/listctrl.cpp

```diff
@@ -5542,6 +5542,15 @@ wxString wxGenericListCtrl::OnGetItemText(long WXUNUSED(item), long WXUNUSED(col
     return wxEmptyString;
 }
 
+bool wxGenericListCtrl::OnGetItemIsChecked(long WXUNUSED(item)) const
+{
+    // this is a pure virtual function, in fact - which is not really pure
+    // because the controls which are not virtual don't need to implement it
+    wxFAIL_MSG(wxT("wxGenericListCtrl::OnGetItemIsChecked not supposed to be called"));
```

Comment:
```suggestion
    wxFAIL_MSG(wxS("wxGenericListCtrl::OnGetItemIsChecked not supposed to be called"));
```
Or just remove it entirely. But there is really no good reason to use `wxT` in the new code.